[2025-03-27T20:32:46.002+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:32:38.921082+00:00 [queued]>
[2025-03-27T20:32:46.023+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:32:38.921082+00:00 [queued]>
[2025-03-27T20:32:46.024+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-27T20:32:46.054+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transfer_data> on 2025-03-27 19:32:38.921082+00:00
[2025-03-27T20:32:46.061+0100] {standard_task_runner.py:60} INFO - Started process 24665 to run task
[2025-03-27T20:32:46.070+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag2', 'transfer_data', 'manual__2025-03-27T19:32:38.921082+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/complete.py', '--cfg-path', '/tmp/tmpggdgdbvm']
[2025-03-27T20:32:46.075+0100] {standard_task_runner.py:88} INFO - Job 121: Subtask transfer_data
[2025-03-27T20:32:46.224+0100] {task_command.py:423} INFO - Running <TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:32:38.921082+00:00 [running]> on host LAPTOP-A4DHQTS9.
[2025-03-27T20:32:46.484+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag2' AIRFLOW_CTX_TASK_ID='transfer_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T19:32:38.921082+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-27T19:32:38.921082+00:00'
[2025-03-27T20:32:46.525+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-27T20:32:46.553+0100] {sql.py:450} INFO - Running statement: SELECT COALESCE(MAX(lastupdated), '1900-01-01'::timestamp) FROM staging.stg_prog3;, parameters: None
[2025-03-27T20:32:46.557+0100] {sql.py:459} INFO - Rows affected: 1
[2025-03-27T20:32:46.559+0100] {logging_mixin.py:188} INFO - Latest lastupdated in staging: 1900-01-01 00:00:00
[2025-03-27T20:32:46.584+0100] {base.py:83} INFO - Using connection ID 'source_db' for task execution.
[2025-03-27T20:32:46.602+0100] {sql.py:450} INFO - Running statement: 
        SELECT * FROM prog3 
        WHERE lastupdated > '1900-01-01 00:00:00';
    , parameters: None
[2025-03-27T20:32:46.609+0100] {sql.py:459} INFO - Rows affected: 30
[2025-03-27T20:32:46.642+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-27T20:32:46.692+0100] {sql.py:547} INFO - Done loading. Loaded a total of 30 rows into staging.stg_prog3
[2025-03-27T20:32:46.694+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-03-27T20:32:46.740+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_dag2, task_id=transfer_data, execution_date=20250327T193238, start_date=20250327T193246, end_date=20250327T193246
[2025-03-27T20:32:46.805+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-27T20:32:46.867+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
